src/common/ZulipSwitch.js

```diff
@@ -24,14 +25,16 @@ export default class ZulipSwitch extends PureComponent<Props> {
 
   render() {
     const { disabled, onValueChange, value } = this.props;
-
     return (
       <Switch
         value={value}
         trackColor={{
           false: 'hsl(0, 0%, 86%)',
-          true: BRAND_COLOR,
+          true: Color(BRAND_COLOR)
+            .fade(0.7)
+            .toString(),
         }}
+        thumbColor={value ? BRAND_COLOR : '#f1f1f1'}
```

Comment:
> Ah, but also: Where does #f1f1f1 for the "off" position come from? 🙂

It is a bad fix, but I had no other option as 'unset' or 'none' aren't working at all for the thumbColor prop.
What I mean by that is:
![Screenshot from 2021-04-06 23-37-09](https://user-images.githubusercontent.com/64399555/113758188-3139df00-9731-11eb-9a04-742b6236ce23.png)
The thumbColor sets to the available value (even when it isn't true) whenever an invalid value is set. This isn't an issue with the boolean `value` as it appears to be working whenever we set a valid color for the false condition (just like now, #f1f1f1 is applied when the `value` equals false. And I didn't find any default option for this prop on the internet as well.

I also tried using null and 0, but that lead to removing the thumb as a whole :slightly_smiling_face:  (whenever it is switched off)

Reply:
> (I didn't notice any of this in my earlier reviews, since at that point I'd done most of my manual testing on iOS.)

I'm curious about how this looks on iOS now.

Reply:
Ah, OK, got it.

I think Material Design would have us using its `colorSurface` value. See "Color" in the table under "Thumb attributes" in [this Material Design doc](https://material.io/components/switches/android#anatomy-and-key-properties). (Huh, and it would have us using a secondary color, instead of the primary color, for the "checked" state. I guess we can worry about that later, if at all.)

But there's quite a bit of [setup](https://material.io/develop/android/docs/getting-started) involved if we want to go farther down the paths Material Design would have us take. In particular, I think that work would be required if we want to hook into a `colorSurface` value the right way.

Checking to see if we have a variable that we use roughly the same way as `colorSurface`, I think it might be `backgroundColor`, from the ThemeContext. Here's what a [Material Design doc](https://material.io/develop/android/theming/color) says about `colorSurface`: "Surface colors affect surfaces of components, such as cards, sheets, and menus." Not too descriptive, I guess. I believe (from [this doc](https://material.io/develop/android/theming/dark)) that `colorSurface` varies between light and dark modes, and our `backgroundColor` does that too.

OK, here's what it looks like with `backgroundColor`, since I was curious and tried it out. In light and dark modes, on Android:

<img width=300 src="https://user-images.githubusercontent.com/22248748/113762740-ea26fc00-96e6-11eb-94e5-8dacb88f23ff.png" /><img width=300 src="https://user-images.githubusercontent.com/22248748/113762777-f3b06400-96e6-11eb-8e1a-4df31b882dda.png" />

Hmm. I'm actually not so sure I like how it looks in dark mode; the thumb looks too dark, to me. But I'm not a regular Android user. Would you see that and think it looks like what other apps do?

@gnprice, should I open an issue for following [this setup](https://material.io/develop/android/docs/getting-started) to allow a deeper connection to Material Design color patterns on Android? I'm even less familiar with a comparable setup for iOS, but I think the place to start there would be [this doc](https://material.io/develop/ios).

Reply:
>  the thumb looks too dark, to me.

Yes, it is definitely looking odd.

> Would you see that and think it looks like what other apps do?

Seems like everyone is using a light color for the thumb...

Discord:
<p float="left">
  <img src="https://user-images.githubusercontent.com/64399555/113767418-48ca9500-973c-11eb-9543-404916250d65.jpg" width="250">
  <img src="https://user-images.githubusercontent.com/64399555/113767411-4700d180-973c-11eb-937b-9742964dd83b.jpg" width="250">
</p>
Slack:
<p float="left">
  <img src="https://user-images.githubusercontent.com/64399555/113767404-449e7780-973c-11eb-90ca-50236a4f4fc9.jpg" width="250">
  <img src="https://user-images.githubusercontent.com/64399555/113767410-46683b00-973c-11eb-9035-dfcfaba1358f.jpg" width="250">
</p>
Rocket.Chat:
<p float="left">
  <img src="https://user-images.githubusercontent.com/64399555/113767422-49632b80-973c-11eb-96f2-2f3da61fc3fa.jpg" width="250">
  <img src="https://user-images.githubusercontent.com/64399555/113767425-49fbc200-973c-11eb-996f-53c6e4a414f0.jpg" width="250">
</p>

Reply:
Thanks for grabbing all those screenshots; it's really helpful to see what other apps do!

Yes, I agree; I think a light "thumb" looks better, even in dark mode. Could go with `#FFFFFF`, since that's `colorSurface`'s default value in light mode. Then have a code comment mentioning `colorSurface`, in case we get around to using such a thing. For example, something like

```js
        thumbColor={
          /* eslint-disable operator-linebreak */
          value
            ? // Material design would actually have this be a secondary
              // color, not a primary color. See "Thumb attributes" at
              // this doc:
              //   https://material.io/components/switches/android#anatomy-and-key-properties
              BRAND_COLOR
            : // Material design would have this be `colorSurface`
              // (see above-linked doc), which defaults to `#FFFFFF`,
              // at least in light mode; see
              //   https://material.io/develop/android/theming/color.
              '#FFFFFF'
        }
```

Reply:
Done!